yarn run v1.15.2
$ node local-service.js
UNCAUGHT EXCEPTION
[Inside 'uncaughtException' event] Error: listen EADDRINUSE: address already in use :::8081
    at Server.setupListenHandle [as _listen2] (net.js:1270:14)
    at listenInCluster (net.js:1318:12)
    at Server.listen (net.js:1405:7)
    at Function.listen (/Users/jonathan.cardenas/eosio-toppings/node_modules/express/lib/application.js:618:24)
    at Object.<anonymous> (/Users/jonathan.cardenas/eosio-toppings/packages/api-eosio-compiler/local-service.js:20:5)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
Done in 0.38s.
